### PR TITLE
Pin setuptools for Travis (SC-1136)

### PR DESCRIPTION
**Pull request by TheRealFalcon**
```
Pin setuptools for Travis

Pinned to the latest version that works across versions.
See https://github.com/pypa/setuptools/issues/3118
```

### CODE REVIEW

**Review by paride — APPROVED** [review]
+1, my only comment is that I find https://github.com/pypa/setuptools/blob/main/CHANGES.rst#v5970= a better reference to justify the specific pinned version, as the linked setuptools issue doesn't seem to reference it.

**Review by holmanb — APPROVED** [review]
I'm fine with pinning the same for all versions, but perhaps a comment about which tests fail (3.7/3.8) will be appreciated in the future for context? I guess we should be able to just mine this PR on Github for context, so whether or not you want to add a comment, LGTM.

**Comment by TheRealFalcon** [comment]
Added an additional comments about python 3.7 and 3.8. I'll merge this then if tests pass.

**Pull request merged**
